api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -362,14 +362,30 @@ message HeaderMutation {
   repeated string remove_headers = 2;
 }
 
-// Replace the entire message body chunk received in the corresponding
-// HttpBody message with this new body, or clear the body.
+// The body response message corresponding to FULL_DUPLEX_STREAMED body mode.
```

Comment:
Not implemented hide here.

If you want to wait for load test before unhide this, then we should hide all.

Reply:
Ok, let me hide them all for now.
The benefit of hid them all at this moment is that if we found issues during load test, it will be much easier to change the API.